packages/shopify-codemod/transforms/remove-redundant-returns-from-addeventlistener.js

```diff
@@ -0,0 +1,28 @@
+export default function transformer(file, api) {
```

Comment:
Please unpack `j`, `source`, etc using the same style as our other transforms, e.g.,:

`export default function functionToArrow({source}, {jscodeshift: j}, {printOptions = {}}) {`